ignite/templates/app/files/app/app.go.plush

```diff
@@ -815,13 +814,14 @@ func (app *App) RegisterAPIRoutes(apiSvr *api.Server, apiConfig config.APIConfig
 	authtx.RegisterGRPCGatewayRoutes(clientCtx, apiSvr.GRPCGatewayRouter)
 	// Register new tendermint queries routes from grpc-gateway.
 	tmservice.RegisterGRPCGatewayRoutes(clientCtx, apiSvr.GRPCGatewayRouter)
+	// Register node gRPC service for grpc-gateway.
+	nodeservice.RegisterGRPCGatewayRoutes(clientCtx, apiSvr.GRPCGatewayRouter)
```

Comment:
Why do we need this call plz?

Reply:
https://github.com/cosmos/cosmos-sdk/blob/v0.46.7/client/grpc/node/service.go
https://github.com/cosmos/cosmos-sdk/tree/v0.46.7/client/grpc/node

This call registers the gRPC service for node information, which is something we should include in the template.  This addition has nothing to do with the `pkg/openapiconsole` logic, should I just move it to another PR?

Reply:
Not necessarily an other PR, but at least a second CL entry, something like *register new GRPC service for querying chain config*.